src/material-examples/autocomplete-display/autocomplete-display-example.ts

```diff
@@ -31,8 +31,8 @@ export class AutocompleteDisplayExample {
   ngOnInit() {
     this.filteredOptions = this.myControl.valueChanges
       .pipe(
-        startWith({} as User),
-        map(user => user && typeof user === 'object' ? user.name : user),
+        startWith<string | User>(this.myControl.value),
```

Comment:
`myControl.value` will always be undefined. You could start it off with an empty string instead.